client/src/js/components/bhPatientFinancialActivity.js

```diff
@@ -59,6 +59,16 @@ function PatientFinancialActivityCtrl(Patients, moment, Session, Constants) {
       .finally(() => {
         $ctrl.loading = false;
       });
+
+    Patients.getStockMovements($ctrl.patientUuid)
+      .then(data => {
+        $ctrl.dataMovement = data;
+        $ctrl.noStockMovement = data.length === 0;
```

Comment:
If there is no financial activity (but there is stock activity), this component will be covered up with a message.  Here is an example:
![image](https://user-images.githubusercontent.com/896472/78898904-7cf97000-7a6c-11ea-9b39-00ecb25bfe63.png)

There is stock data here, but no financial data.  Could you change [this line](https://github.com/IMA-WorldHealth/bhima/pull/4269/files#diff-8422e147a296cedc27c69a1710e1cd2aL21) from:
```html
<bh-panel-overlay ng-if="$ctrl.noFinancialHistory">
```
to
```html
<bh-panel-overlay ng-if="$ctrl.noFinancialHistory && $ctrl.noStockMovement">
```